### ISSUE TITLE
Training Loss

### ISSUE THREAD

**Issue by jinan-zhou** [Garrafao/TemporalReferencing#3]
Is that a way to print out the the loss when training words embeddings? It may help to determine the right number of iterations needed.

**Comment by Garrafao** [comment]
I don't know. I think the best address to ask for this would be the creators of [hyperwords](https://bitbucket.org/omerlevy/hyperwords/src/default/). Hope this helps.
